pkg/controller/controller_test.go

```diff
@@ -138,7 +138,7 @@ func (s *ControllerSuite) TestWatch(c *C) {
 
 // nolint:unparam
 func (s *ControllerSuite) waitOnActionSetState(c *C, as *crv1alpha1.ActionSet, state crv1alpha1.State) error {
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
Is this intentional?

Reply:
yes. I can see if I can make it to 20 to 15.

Reply:
I will make this change as part of test PR, reverting this back to what it was earlier.